sdk/src/test/java/com/finbourne/lusid/tutorials/ibor/Valuation.java

```diff
@@ -281,7 +281,12 @@ private void run_aggregation(
         ConfigurationRecipe configurationRecipe = new ConfigurationRecipe()
                 .code("quotes_recipe")
                 .market(new MarketContext()
-                        .suppliers(new MarketContextSuppliers().equity(MarketContextSuppliers.EquityEnum.DATASCOPE))
+                        // equity data and fx data are sourced from different data providers, reference the createUpsertQuote... methods
+                        // to view how the quote requests are mapped to providers.
+                        .suppliers(new MarketContextSuppliers()
+                                .equity(MarketContextSuppliers.EquityEnum.DATASCOPE)
```

Comment:
my fault I probably wasn't clear in my last comment, could we change DataScope to LUSID unless we intentionally need to show 2 different providers?

Reply:
Done. All test cases now only use Lusid data provider.